backend/src/main/java/com/now/naaga/player/presentation/dto/RankResponse.java

```diff
@@ -0,0 +1,40 @@
+package com.now.naaga.player.presentation.dto;
+
+import com.now.naaga.player.domain.Player;
+
+public class RankResponse {
+
+    private PlayerResponse player;
+    private int rank;
+    private int topPercent;
+
+    public RankResponse() {
+    }
+
+    public RankResponse(final PlayerResponse player,
+                        final int rank,
+                        final int topPercent) {
+        this.player = player;
+        this.rank = rank;
+        this.topPercent = topPercent;
+    }
+
+    public static RankResponse to(final Player player,
```

Comment:
```suggestion
    public static RankResponse of(final Player player,
```
이전의 리뷰와 같은 맥락으로, 이 경우에는 of 가 적절한 것 같습니다.

Reply:
to 는 반환값이 변형될때, of는 반환값이 같을때 사용하네요.. 수정하였습니다!